custom-actions/timeline/Extreme_Reach_IO.py

```diff
@@ -0,0 +1,118 @@
+#Script to set in and out points in a timeline based on the Extreme Reach delivery specs.
+def get_media_panel_custom_ui_actions():
+
+    def scope_clip(selection):
+        import flame
+        for item in selection:
+            if isinstance(item, (flame.PyClip)):
+                return True
+        return False
+
+    def er_io_60(selection):
+        import flame
+        for sequence in selection:
+            i = sequence.duration.frame
+            frameRate = sequence.frame_rate
+            tc_IN = flame.PyTime("00:59:53:00", frameRate)
```

Comment:
Could assign directly sequence.in_mark and sequence.out_mark. No need for a intermediary variable.

Reply:
Do you mean that I don't need line 14:   i = sequence.duration.frame  There's a very good chance that this line was left in during my testing phase, when I had "print I" in the script so I could see the length of the sequence in the console.

Reply:
Ah...nvm. I tried commenting it out and see that it was a leftover from my testing phase. Removed!

Reply:
```suggestion
            sequence.in_mark = flame.PyTime("00:59:53:00", frameRate)
            sequence.out_mark = flame.PyTime("01:01:00:01", frameRate)
```